lib/thrift/binary/framed/server.ex

```diff
@@ -2,11 +2,18 @@ defmodule Thrift.Binary.Framed.Server
   do
   @moduledoc false
 
+  @type tcp_options :: [
+    recv_timeout: timeout,
+    timeout: timeout,
```

Comment:
Should this become `conn_timeout` (or some other prefix) now that we also have `{recv,send}_timeout` alongside it? Or does this need to match ranch/gen_tcp?

Or is this unused and we only use `:ranch_tcp.opts` below?